spring-boot-project/spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -859,18 +921,23 @@ The example in the previous section can be rewritten in an immutable fashion as
 
 		private final InetAddress remoteAddress;
 
+		private final int port;
+
 		private final Security security;
 
-		public AcmeProperties(boolean enabled, InetAddress remoteAddress, Security security) {
+		public AcmeProperties(boolean enabled, InetAddress remoteAddress, int port, Security security) {
```

Comment:
added port to enable instantiation of the server in https://github.com/spring-projects/spring-boot/pull/18017/files#diff-51b04ca54a1bd6f28cae0525b2b1a7fdR23